templates/common/.vscode/setup-web.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
@philliphoff @danieljurek Did we land on what the long-term plan for codespaces support is ? I'm a little worried about the side-effects of each service modifying the `env` file altogether. It might result in unintended behaviors. For example, things that work when only API runs, when only Web runs, or when both of them run.

Piecing together in my mind what the requirements are for codespaces:
1. The front-end web app needs `REACT_APP_API_BASE_URL` set to the public endpoint.
2. The back-end web app port-forwarding config needs to be set to Public visibility. Otherwise, we're hit with CORS errors by gh's reverse proxy as they do not share origins.

As a solution to 1, I wonder if we can use an input command, one that looks like:
```
command: "azure-dev.resolve-dev-endpoint",
args: {
  defaultHostName: "localhost",
  defaultPort: "3000"
}
```

that will resolve the hostname to either the codespaces URL or localhost depending on where we're running. It would also set some input variable to indicate "running in codespaces"

As a solution to 2, we could have a post API run script that configures the visibility. I do wonder if there's a better alternative here than publicly exposing the API.

Reply:
https://github.com/Azure/azure-dev/issues/918

We were heading down the path of using an input of sorts to specify a URL in the environment.